spec/lib/evss/pciu_address/service_spec.rb

```diff
@@ -0,0 +1,71 @@
+# frozen_string_literal: true
+require 'rails_helper'
+
+describe EVSS::PCIUAddress::Service do
+  let(:user) { build(:loa3_user) }
+
+  describe '#get_countries' do
+    context 'with a 200 response' do
+      it 'returns a list of countries' do
+        VCR.use_cassette('evss/pciu_address/countries') do
```

Comment:
I've sometimes found it difficult to regenerate someone else's VCR cassettes.  I try to leave a comment if regeneration requires more than just deleting the cassette and re-running.  I'm not sure if more than that is the case here, or if leaving a comment is even the best solution, but it's probably better than nothing.